### PR TITLE
version test case expects 500 if version file is not found

### PR DESCRIPTION
**Pull request by DrFaustie**
Fixes #1860 
Test cases should now reflect changes

### CODE REVIEW

**Comment by DrFaustie** [comment]
Travis CI seems to still be failing, I'm looking at it now. But can someone please advise me on how to update this pull request instead of spamming with more new ones?

**Comment by leplatrem** [comment]
> Travis CI seems to still be failing, I'm looking at it now. 

you should run this locally before relying on TravisCI:  

`$ py.test tests/core/test_views_version.py`

> But can someone please advise me on how to update this pull request instead of spamming with more new ones?

Just push more commits to your master branch and it will update this PR automatically. Close the ones that you want to ignore ;)

**Comment by DrFaustie** [comment]
Thank you

**Comment by Natim** [comment]
Here it seems you lost your previous commit...

**Comment by Natim** [comment]
Nice job :+1:

**Comment by DrFaustie** [comment]
Can't believe I made that mistake! I'm still very new to pull requests so apologies for the mistakes :>

**Comment by Natim** [comment]
No worries, we all started messing around with Git, it gets better with experience

**Comment by Natim** [comment]
> kinto/core/views/version.py:5:1: F401 'pyramid.httpexceptions' imported but unused

You can remove the unused import as well

**Comment by DrFaustie** [comment]
Regarding the changelog, should I just add the change **11.2.0 (unreleased)**?

**Comment by DrFaustie** [comment]
Thank you for holding my hand through this!

File: kinto/core/views/version.py [hunk 1]
```diff
@@ -46,4 +45,4 @@ def version_view(request):
                 version_view.__json__ = json.load(f)
                 return version_view.__json__  # First one wins.
 
-    raise httpexceptions.HTTPNotFound()
+    raise FileNotFoundError("Version file missing")
```
[the remark this answers is not in the document]
Comment (DrFaustie) [reply kinto/core/views/version.py]: Perhaps `raise FileNotFoundError("Version file missing from {}".format(files.join(",")))` can be used here too?

**Comment by leplatrem** [comment]
Thanks a lot! Welcome in the game ;)

**Pull request merged**